source/dotnet/Samples/AdaptiveCards.Sample.Html/Program.cs

```diff
@@ -147,6 +147,7 @@ static int Main(string[] args)
                     }
                     catch (Exception err)
                     {
+                        Debugger.Break();
```

Comment:
>Debugger.Break(); [](start = 24, length = 17)

wasn't this line just removed in another PR?

Reply:
it was removed in one of the PRs I'm reverting. I'll get rid of it again :)

---
In reply to: [312569440](https://github.com/microsoft/AdaptiveCards/pull/3325#discussion_r312569440) [](ancestors = 312569440)